README.md

```diff
@@ -82,6 +82,12 @@ jobs:
 ```yaml
 - uses: nrwl/ci/.github/workflows/nx-cloud-main.yml@v0.3
   with:
+    # [OPTIONAL] The available number of agents used for fine tuning
+    # the Nx Cloud to intelligently distribute tasks in parallel.
+    #
+    # Default: 6
```

Comment:
Should we remove the default (provided you agree with the conversation above)?

and let me know what you think about an adjust to the comment like this:

```
    # [OPTIONAL] By default, NxCloud tries to infer dynamically how many agents you have available for task distribution. If you know exactly how many agents you always have available, it is recommended to set this so it can more reliably distribute the tasks.
```

Reply:
Added more text. Thanks!